robobreizh/GraspServerRequest request
---
robobreizh/GraspConfigList grasp_configs
